tinygrad/jit.py

```diff
@@ -3,21 +3,26 @@
 from collections import defaultdict
 import functools, itertools
 from tinygrad.helpers import DEBUG, DType, merge_dicts
-from tinygrad.ops import Device
+from tinygrad.ops import RawBuffer, Device, Compiled
 from tinygrad.tensor import Tensor
-from tinygrad.ops import RawBuffer
 from tinygrad.shape.shapetracker import ShapeTracker
 from tinygrad.shape.symbolic import Variable
 
 JIT_SUPPORTED_DEVICE = ["GPU", "CLANG", "METAL", "CUDA", "HIP", "WEBGPU", "LLVM"]
 
+class JitCtx:
```

Comment:
Does this have to be a separate class? I don't see why this has to change

Reply:
Removed it